libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -955,8 +955,9 @@ void AP_DDS_Client::on_request(uxrSession* uxr_session, uxrObjectId object_id, u
             .type = UXR_REPLIER_ID
         };
 
-        uint32_t reply_size = rcl_interfaces_srv_SetParameters_Response_size_of_topic(&set_parameter_response, 0U);
-        uint8_t reply_buffer[reply_size] {};
+        const uint32_t reply_size = rcl_interfaces_srv_SetParameters_Response_size_of_topic(&set_parameter_response, 0U);
+        uint8_t reply_buffer[reply_size];
```

Comment:
Is it any initialisation of variable length arrays, or this specific syntax? Can you do the following instead?

```suggestion
        uint8_t reply_buffer[reply_size] = { 0 };
```

Reply:
No. The suggested change fails to compile.

```bash
../../libraries/AP_DDS/AP_DDS_Client.cpp:967:30: fatal error: variable-sized object may not be initialized
        uint8_t reply_buffer[reply_size] = { 0 };
                             ^~~~~~~~~~
2 warnings and 1 error generated.
```